src/main.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * 
+ * main.c
+ *  
+ */
+
+#include <stdio.h>
+#include "xil_printf.h"
+#include "xil_io.h"
+#include "xparameters.h"
+#include "controller.h"
+
+
+int main()
+{
+	printf("Program Start.\n");
+	
+	// BRAM1[3] <= BRAM0[0] * BRAM1[2] ;
+	// BRAM1[7] <= BRAM0[11] * BRAM1[3] ;
+	// BRAM1[10] <= BRAM0[31] * BRAM1[7] + C ;
+	// BRAM1[13] <= C - BRAM0[1] * BRAM1[6] ;
+	// BRAM1[15] <= BRAM0[0] * BRAM1[31] - C - 1 ;
+
+	u32 data1, data2, result;
+
+	
+	// alumode = 0000, opmode = 0000101, inmode = 10001, bram1wAddr = 4*3, bram1rAddr = 4*2, bram0rAddr = 4*0;
+	exec(XPAR_CONTROLLER_0_S00_AXI_BASEADDR,
+		 codegen(0, 0x0000101, 0x10001, 12, 8, 0)); 

```

Comment:
`0x0000101` and `0x10001` are hexdecimal numbers. Should it be `0b000101` and `0b10001`?
Besides, `bram1waddr`, `bram1raddr`, and `bram0raddr` are all **word** address, which in the other words we don't have to multiply them by 4. You may think of it as the same concept of **register number** in CPU ISAs.